sdk/formrecognizer/Azure.AI.FormRecognizer/tests/DocumentAnalysisClient/DocumentAnalysisClientMockTests.cs

```diff
@@ -202,6 +203,86 @@ public async Task AnalyzeDocumentFromUriSendsMultiplePageArgument(string page1,
             Assert.True(requestUriQuery.Contains(expectedSubstring));
         }
 
+        [Test]
+        public async Task AnalyzeDocumentSendsSingleFeature()
```

Comment:
We do not have test files we can make public to test any of these features, so I'm adding mock tests for now. They have been live tested locally, though, and all of them work.